ui/src/main/resources/FileManagerCode/RecentFilesMacro.xml

```diff
@@ -308,7 +321,18 @@
   #set ($fileDoc = $xwiki.getDocument($result))
   #if ($fileDoc)
     #getFile($fileDoc $file)
-    * {{html}}#mimetypeimg($file.mediaType.toLowerCase() $file.name.toLowerCase()){{/html}} [[$file.name&gt;&gt;$result]] $services.localization.render('fileManager.recentFiles.creator', ["[[$xwiki.getUserName($file.creator, false)&gt;&gt;$file.creator]]"]) $services.localization.render('fileManager.recentFiles.creationDate', [$xwiki.formatDate($file.creationDate)])
+    #getFileFolders($fileDoc $paths)
+    #set ($fileUrl = $fileDoc.getURL('view'))
+    #set ($creatorUrl = $xwiki.getDocument($file.creator).getURL('view'))
+    * {{html}}
+      #mimetypeimg($file.mediaType.toLowerCase() $file.name.toLowerCase())
+      &lt;a href="$fileUrl" title="$folders"&gt;$file.name&lt;/a&gt;
+      &lt;span&gt;$services.localization.render('fileManager.recentFiles.creator')&lt;/span&gt;
+      &lt;a href="$creatorUrl"&gt;$xwiki.getUserName($file.creator, false)&lt;/a&gt;
+      &lt;span&gt;
+        $services.localization.render('fileManager.recentFiles.creationDate', [$xwiki.formatDate($file.creationDate)])
+      &lt;/span&gt;
```

Comment:
You should always escape / encode special XML characters that come from user input, including translations (we don't always review translations thoroughly).